src/Components/test/testassets/ComponentsApp.Server/ComponentsApp.Server.csproj

```diff
@@ -6,7 +6,6 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore" />
-    <Reference Include="Microsoft.AspNetCore.Mvc.Components.Prerendering" />
     <Reference Include="Microsoft.AspNetCore.Components.Server" />
     <Reference Include="Microsoft.AspNetCore.Mvc" />
     <Reference Include="Newtonsoft.Json" />
```

Comment:
Unrelated, but what's this for?

Reply:
JSON.NET? I think it's for json serialization.